src/components/Partners/MainPartner.js

```diff
@@ -0,0 +1,56 @@
+import { A } from "@commons-ui/core";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+const useStyles = makeStyles(({ breakpoints, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(44.41),
+    minHeight: typography.pxToRem(300),
+    [breakpoints.up("md")]: {
+      minHeight: typography.pxToRem(290),
+    },
+    [breakpoints.up("lg")]: {
+      paddingTop: typography.pxToRem(46.07),
+      paddingLeft: typography.pxToRem(119),
+    },
+  },
+  link: {
+    display: "block",
+  },
+  description: {
+    marginTop: typography.pxToRem(40.01),
+  },
+}));
+
+function MainPartner({ link, logo, name, description, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root} item xs={12} md={6}>
```

Comment:
Warning: Received `true` for a non-boolean attribute `item`.

Was this meant to be a `Grid`?